jpa-persistence-lib-product/README.md

```diff
@@ -0,0 +1,13 @@
+# JPA Persistence Lib Demo
```

Comment:
@andreasbalsiger please review the following public readme of this new market product.

Btw, I would remove "demo" from the title as a product and a demo are provided.

Reply:
@ivy-lmu, @majeed-hm: To provide more business-like features, I need to know a bit more information about this new product. Please provide me with that infos, maybe you can have a look at this connecter to get inspired: https://developer.axonivy.com/market/ldap-connector
What is the addon to the existing ivy persistence API? Otherwise, a potential user is kind of lost. thanks a lot,
Andreas

Reply:
here's the existing doc, that may contribute to the praise on the market too:
https://axonivy.atlassian.net/wiki/spaces/AIGUIDE01/pages/3311013887/Get+started+with+Standard+Persistency+Library
I think we should copy and adapt this for a larger audience.

Reply:
@andreasbalsiger The Demo project is using the Axon.Ivy persistence-utils and showing how the meta model is generated by maven. Also some Bean Validation is implemented and JUnits with mocked Ivy can be started in the test project either with maven or with the designer.

Reply:
@andreasbalsiger please find below the three files that need to be reviewed or extended. Thank you 

- https://github.com/axonivy-market/persistence-utils/blob/master/README.md
- https://github.com/axonivy-market/persistence-utils/blob/master/persistence-utils-product/README.md
- https://github.com/axonivy-market/market/blob/persistence-utils/market/utils/persistence-utils/meta.json